app/css/pages/mentoring/automation.css

```diff
@@ -0,0 +1,69 @@
+@import "../../styles";
+
+.c-mentor-inbox {
```

Comment:
You're globally changing `c-mentor-inbox` here. You should either:
1. Make these changes in the component file (app/css/pages/mentoring/inbox.css) or 
2. If you want these only for automation, you should add a subselector here:

```.c-mentor-inbox.--automation {```

Then have the class for automation as class: `c-mentor inbox --automation`. 

3. Alternatively you could duplicate the c-mentor-inbox entirely here and change the class.

(1) or (2) are probably right depending on your purpose.